src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonaftertypememberdeclaration/InputUnnecessarySemicolonAfterTypeMemberDeclarationNullAst.java

```diff
@@ -0,0 +1,24 @@
+/*
+UnnecessarySemicolonAfterTypeMemberDeclaration
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, VARIABLE_DEF, \
+         ANNOTATION_FIELD_DEF, STATIC_INIT, INSTANCE_INIT, CTOR_DEF, METHOD_DEF, \
+         ENUM_CONSTANT_DEF, COMPACT_CTOR_DEF, RECORD_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonaftertypememberdeclaration;
+
+// ok
+public class InputUnnecessarySemicolonAfterTypeMemberDeclarationNullAst {
+    public static void main(String[] args) {
+        switch (args.length) {
+            case 0:
+                final int k = 12;
+                class Local {
+                    int j = k;
+                }
```

Comment:
Please add test cases as follows:
1. Add another local class declaration to this block statement WITH an unnecessary semicolon.
2. Add another local class with fields that have extra semicolons, to show that we still place violations on them. See the check documentation for some examples.